Exercise1.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+"""
```

Comment:
The docstring should be inside the function itself.

See https://www.python.org/dev/peps/pep-0257/ for details.

If should probably look like:

```python
  def is_permutation(string1, string2):
    """Determines if two strings are permutations of each other.

    Args:
      string1: the first string to compare
      string2: the second string to compare

    Returns:
       True if the two strings are permutations of each other, otherwise False.
    """
```